spec/models/utils/load_testing/odk_submission_load_test_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Utils::LoadTesting::OdkSubmissionLoadTest do
+  let(:mission) { FactoryGirl.create(:mission, name: "ODK Submission Load Test Mission") }
+  let(:form) do
+    FactoryGirl.create(:form, mission: mission, question_types: %w[
+      text
+      long_text
+      integer
+      counter
+      decimal
+      location
+      select_one
+      select_multiple
+      datetime
+      date
+      time
+      barcode
+    ])
+  end
+
+  let(:username) { "admin" }
+  let(:password) { "Testing123" }
+  let(:threads) { 1 }
+  let(:duration) { 5 }
+
+  before { form.publish! }
+
+  it "generates expected test plan" do
+    test = described_class.new(
+      threads: threads,
+      duration: duration,
+      username: username,
+      password: password,
+      mission_name: mission.compact_name,
+      form_id: form.id
+    )
+
+    path = test.generate
+    actual_content = File.read(path.join("testplan.jmx"))
+    expected_content = fixture_file("test_plans/odk_submission.jmx")
+
+    expect(without_timestamps(actual_content)).to eq(without_timestamps(expected_content))
```

Comment:
This is super cool! If my request to use CSV with XML causes this method to break somewhat, I wouldn't stress out too much and it doesn't have to test every last char of the test plan. The main thing is that we are just running the generating the test plan in the spec suite and not getting any errors.